### ISSUE TITLE
status of project

### ISSUE THREAD

**Issue by iDoka** [autowp/CANreader-XL#1]
Hello
That's finished project? Is FW (for STM32) available?
Thanks

**Comment by autowp** [comment]
Nope. Dropped on trying create multi USB device firmware CDC+CDC+HID

чт, 10 нояб. 2016 г., 23:33 Dmitry Murzinov notifications@github.com:

**Comment by iDoka** [comment]
I think dual CDC not so difficult as well:
https://github.com/rgwan/libopencm3-examples/tree/63d52f65e2271aa1eef4d588be748e9917ebed23/examples/stm32/f1/stm32-h103/usb_dualcdc
https://github.com/x893/STM32F103-DualCDC

For what purposes do you need HID?
Probably use out of band command thru CDC?

**Comment by autowp** [comment]
For enable media buttons on steering wheel

пт, 11 нояб. 2016 г., 8:51 Dmitry Murzinov notifications@github.com:
